### PR TITLE
[mds-core] Onboarding Improvements

### PR DESCRIPTION
**Pull request by kemiller**
Some additions to the onboarding docs, and making kubernetes/docker integration tests a little less specific so they should pass for anyone if everything is working.

No code impact.

### CODE REVIEW

**Comment by kemiller** [comment]
@jwtodd I changed a test you wrote.

**Comment by jwtodd** [comment]
@kemiller nice! i started some work to make that test green (again) which led to some re-imagining work @macsj200 and i are collaborating on [ see feature/maxj/autogen-integration-spike , https://github.com/lacuna-tech/mds-core/blob/feature/maxj/autogen-integration-spike/packages/mds-agency/tests/agency-about.ispec.ts ]

that aside and till we get this work furthered (in the coming days/weeks) this is a good patch imo

File: README.md [hunk 1]
```diff
@@ -33,6 +34,15 @@ If you encounter the following error:
 The following command should fix your issue
 `createdb -h localhost`
 
+To run tests, you will need this:
+`createdb -h localhost lacuna`
```
Comment (avatarneil) [inline README.md]: Seeing as the contributions in this repo are intended to be merged back into OMF at some point, we probably don't want to specify anything about `lacuna` in here. Perhaps `mds`?

Comment (kemiller) [reply README.md]: Gotcha.  Will make this more generic.

**Pull request merged**